README.md

```diff
@@ -105,3 +105,35 @@ When you choose a name other than "editorconfig.json", enter the path of the edi
 ***
 
 After the above work, restart Firefox.
+
+***
+
+## Troubleshooting
+
+If something goes wrong, check the browser console (Ctrl + Shift + J).
+
+```
+Error: Attempt to postMessage on disconnected port
+```
+
+* Windows: Is the registory saved correctly?
+* Linux / Mac: Is "withexeditorhost.json" saved in the right location?
+* When you start Firefox, is Node.js process executed too?
+  * If not, make sure there is $PATH environment variable for Node.js.
+    Or, change shell script from "node" command to full path of Node.js.
+    ```
+    /path/to/node.js /path/to/withexeditorhost/index.js
+    ```
+  * Also, make sure withexeditorhost.sh has executable bit (Linux / Mac).
```

Comment:
"Also, make sure the execute bit is set on withexeditorhost.sh (Linux / Mac)."

or

"Also, make sure that you've marked withexeditorhost.sh as executable (Linux / Mac)."

or 

"Also, make sure that you've set withexeditorhost.sh to be executable (Linux / Mac)."

(It always "has" the bit. It's just a question of what state it's in.)